src/python/py_whisper_pipeline.cpp

```diff
@@ -156,6 +147,46 @@ auto whisper_generation_config_docstring = R"(
+    stop_token_ids: a set of tokens that will cause pipeline to stop generating further tokens.
+    echo:           if set to true, the model will echo the prompt in the output.
+    logprobs:       number of top logprobs computed for each position, if set to 0, logprobs are not computed and value 0.0 is returned.
+                    Currently only single top logprob can be returned, so any logprobs > 1 is treated as logprobs == 1. (default: 0).
+
+    repetition_penalty: the parameter for repetition penalty. 1.0 means no penalty.
+    presence_penalty: reduces absolute log prob if the token was generated at least once.
+    frequency_penalty: reduces absolute log prob as many times as the token was generated.
+
+    Beam search specific parameters:
+    num_beams:         number of beams for beam search. 1 disables beam search.
+    num_beam_groups:   number of groups to divide `num_beams` into in order to ensure diversity among different groups of beams.
+    diversity_penalty: value is subtracted from a beam's score if it generates the same token as any beam from other group at a particular time.
+    length_penalty:    exponential penalty to the length that is used with beam-based generation. It is applied as an exponent to
+        the sequence length, which in turn is used to divide the score of the sequence. Since the score is the log
+        likelihood of the sequence (i.e. negative), length_penalty > 0.0 promotes longer sequences, while
+        length_penalty < 0.0 encourages shorter sequences.
+    num_return_sequences: the number of sequences to return for grouped beam search decoding.
+    no_repeat_ngram_size: if set to int > 0, all ngrams of that size can only occur once.
+    stop_criteria:        controls the stopping condition for grouped beam search. It accepts the following values:
+        "openvino_genai.StopCriteria.EARLY", where the generation stops as soon as there are `num_beams` complete candidates;
+        "openvino_genai.StopCriteria.HEURISTIC" is applied and the generation stops when is it very unlikely to find better candidates;
+        "openvino_genai.StopCriteria.NEVER", where the beam search procedure only stops when there cannot be better candidates (canonical beam search algorithm).
+
+    Random sampling parameters:
+    temperature:        the value used to modulate token probabilities for random sampling.
+    top_p:              if set to float < 1, only the smallest set of most probable tokens with probabilities that add up to top_p or higher are kept for generation.
+    top_k:              the number of highest probability vocabulary tokens to keep for top-k-filtering.
+    do_sample:          whether or not to use multinomial random sampling that add up to `top_p` or higher are kept.
+    num_return_sequences: the number of sequences to generate from a single prompt.
```

Comment:
`rng_seed` is missing. Although it's not Whisper's fault because it's missing in the parent config as well. Can you add it, possible in a separate PR

Reply:
Sure, will do